docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -30,6 +30,12 @@ export default class DatePickerPage extends React.Component {
         name: 'Props',
         infoArray: [
           {
+            name: 'container',
+            type: 'one of: dialog, container',
+            header: 'default: dialog',
+            desc: 'The date pickers container type',
```

Comment:
Much better, but it references "auto complete", which doesn't exist yet. Regardless, I think the description should be standalone. Just "displays the DatePicker below the input field", or perhaps "displays the DatePicker as a drop-down below the input field". @oliviertassinari ?